src/main/java/seedu/smarthomebot/logic/parser/Parser.java

```diff
@@ -0,0 +1,241 @@
+package seedu.smarthomebot.logic.parser;
```

Comment:
DG mentions class diagram. Should be Sequence Diagram instead?
![Uploading Screenshot 2020-10-29 at 16.15.37.png…]()